### DISCUSSION TITLE
Do you name your home gadgets?

### DISCUSSION THREAD

**Discussion by janply** [NAIST-SE/AttractingContributors#735] [Polls]
Do you name your home gadgets?

**Comment by hernessteve** [comment]
Who does that?

**Comment by ellielo** [comment]
Only my vacuum cleaner has a cute name

**Comment by McJul** [comment]
My wife also calls our robotic vacuum cleaner by name. The only gadget I gave name to is my TV, or to be more precise the TV stick. I had to [https://www.firesticktricks.com/change-firestick-name.html](https://www.firesticktricks.com/change-firestick-name.html) because we have a couple of them and need to distinguish them.
